test/test_quantized_models.py

```diff
@@ -25,5 +25,26 @@ def test_float_quant_compare(self):
         # output
         self.assertGreater(SQNRdB, 30, msg='Quantized model numerics diverge from float, expect SQNR > 30 dB')
 
+    def test_float_quant_compare_per_channel(self):
+        # Test for per-channel Quant
+        torch.manual_seed(67)
+        myModel = ModelMultipleOps().to(torch.float32)
```

Comment:
nit: can you use '_' naming e.g. 'my_model', or maybe just 'model'? so that we have a consistent naming?